.gitlab/corona-templates.yml

```
@@ -17,7 +17,7 @@
   rules:
     - if: '$ON_CORONA == "OFF"'
       when: never   # no jobs on corona if ON_CORONA == OFF
-    - if: '$LC_ZONE =~ "CZ"' #run only on CZ ...
+    - if: '$LC_ZONE =~ "RZ"' #run only on CZ ...
```

One again my bad, I appear to be a bit rusted and confused "loose string match" with "not equal":
```suggestion
    - if: '$LC_ZONE != "CZ"' #run only on CZ ...
```